test/org/traccar/FilterHandlerTest.java

```diff
@@ -6,15 +6,19 @@
 import static org.junit.Assert.assertNull;
 import org.junit.Before;
 import org.junit.Test;
+import org.traccar.database.IdentityManager;
 import org.traccar.model.Position;
+import org.traccar.model.Device;
 
 public class FilterHandlerTest {
 
+    private EventHandlerTest deviceHandler;
     private FilterHandler filtingHandler;
     private FilterHandler passingHandler;
 
     @Before
     public void setUp() {
+        deviceHandler = new EventHandlerTest();
```

Comment:
Maybe `BaseTest`. Then `ProtocolTest` can extend it. Actually you can probably rename `EventHandlerTest` to `BaseTest`, and make it a base class of `ProtocolTest`.